src/altinn-app-frontend/src/features/form/components/Panel.test.tsx

```diff
@@ -66,7 +72,7 @@ const render = (
   const allProps = {
     title: 'Panel Title',
     children: 'Panel Content',
-    variant: 'info',
+    variant: 'info' as const,
```

Comment:
Starting at line 72; since allProps does not have a specified type, it is inferred. Since it is a constant (`const allProps`), the property types will be inferred as well. However, it only goes so far - so `variant` is inferred as type `string`. When I tightened the variant property type to be either `'info' | 'warning' | 'success' | 'error'`, giving it a `string` is too broad. By adding 'as const', TypeScript will infer the type of `allProps` to be `{..., variant: 'info'}` not simply `{..., variant: string}`.

In other words, it fixes a TypeScript error.. :-)

Reply:
Nice, learned something new today. Thanks for the explanation.